app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -433,10 +434,65 @@ def test_update_live_rollout(self):
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         self.assertTrue(experiment.is_rollout)
         self.assertEqual(experiment.population_percent, 50.0)
-        self.assertEqual(experiment.publish_status, NimbusConstants.PublishStatus.IDLE)
+        self.assertEqual(experiment.publish_status, NimbusConstants.PublishStatus.DIRTY)
         self.assertEqual(experiment.status, NimbusConstants.Status.LIVE)
         self.assertEqual(experiment.status_next, None)
 
+    @parameterized.expand(
+        [
+            [True, NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE, False],
+            [False, NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_ENROLLING, False],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_ENROLLING, True],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_PAUSED, False],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_PAUSED, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_DIRTY, False],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_DIRTY, True],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_REVIEW_REQUESTED, False],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_REVIEW_REQUESTED, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_REJECT, False],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_REJECT, True],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE, False],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE, False],
+            [False, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_REJECT, False],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_REJECT, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_TIMEOUT, False],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_TIMEOUT, True],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_REJECT_MANUAL_ROLLBACK, False],
+            [True, NimbusExperimentFactory.Lifecycles.LIVE_REJECT_MANUAL_ROLLBACK, True],
+        ]
+    )
+    def test_update_rollout_is_dirty(self, is_dirty_expected, lifecycle, is_paused):
```

Comment:
Testing updating population percentage with a combination of lifecycle and paused status, then seeing what is made dirty

Reply:
Move is_paused back to lifecycles

Reply:
https://github.com/mozilla/experimenter/issues/8334